pkg/middlewares/ratelimiter/redisrate/lua.go

```diff
@@ -0,0 +1,56 @@
+package redisrate
+
+import "github.com/redis/go-redis/v9"
+
+//nolint:dupword
+var allowTokenBucket = redis.NewScript(`
+local key = KEYS[1]
+local limit, burst, ttl, t, n, max_delay = tonumber(ARGV[1]), tonumber(ARGV[2]), tonumber(ARGV[3]), tonumber(ARGV[4]),
```

Comment:
Since we are always going to take 1 request from the bucket, I think we can remove the `n` parameter.

Reply:
If we remove the `n` Parameter, I think I have to remove this function too `AllowTokenBucketN` and replace it with `Allow` only without depending on `n`